platformio.ini

```diff
@@ -25,9 +25,9 @@ build_flags = -DUSE_AUDIO
 lib_deps = dfrobot/DFRobotDFPlayerMini@^1.0.3
 
 [audio_TD5580A]
-build_flags = -DUSE_AUDIO -DUSE_SERIAL_MP3
-lib_deps = 
-	nhluan37/SerialMP3@^1.0.0
+build_flags = -DUSE_AUDIO -DUSE_SERIAL_MP3 -DNO_SERIALMP3_DELAY
+lib_deps =
+	nhluan37/SerialMP3@^1.1.0
```

Comment:
Shouldn't be `nhluan37/SerialMP3@^1.1.0` -> `nhluan37/SerialMP3@1.1.0` with caret sign it will still upgrade it, which I'm not sure is wanted here.

Reply:
The caret means it will pick up the latest patch version (=bug fixes) which is usually desired if the package follows semver.
I've noticed these arduino libraries are quite the wild west when it comes to versioning so I can remove if necessary, I just left it because it was there before.

Reply:
Please remove indeed, we had a fully broken build due to a small update in one of the packages.